staff/aleix-caminal/cv-styled/assets/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  display: grid;
+  grid-template: auto auto/1fr 260px;
+  grid-gap: 1.25rem;
+  padding: 1.25rem;
+  font-family: TT Commons, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Droid Sans, Helvetica Neue;
```

Comment:
Too much fonts were specified.

You need to specify at least the base font type: sans or sans-serif. This charge a font of this type if all the others fail.

Reply:
true xD